lib/sles4sap/ipaddr2.pm

```diff
@@ -373,6 +374,29 @@ sub ipaddr2_deployment_sanity {
     }
 }
 
+=head2 ipaddr2_os_sanity
+
+    ipaddr2_os_sanity()
+
+Run some OS level checks on the various VMs composing the deployment.
+die in case of failure
+
+=over 1
+
+=item B<bastion_ip> - Public IP address of the bastion. Calculated if not provided.
+                      managed as argument not to have to call ipaddr2_bastion_pubip many time,
+                      so not to have to query az each time
+
+=back
+=cut
+
+sub ipaddr2_os_sanity {
+    my (%args) = @_;
+    $args{bastion_ip} //= ipaddr2_bastion_pubip();
+
+    ipaddr2_os_connectivity_sanity(bastion_ip => $args{bastion_ip});
```

Comment:
Are there going to be more sanity checks beside this one ? 
So far it looks like it is not doing much beside  ipaddr2_os_connectivity_sanity()

Reply:
yes, this is the only one is always passing for the moment: a preview of a full list of them is in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19243